client/src/app/shared/loading-bar/loading-bar.component.html

```diff
@@ -0,0 +1,3 @@
+<ng-container *ngIf="_isVisible">
```

Comment:
Any particular reason this is wrapped in an `<ng-container>`?

Also the template of this component is so small, we can get rid off the template and inline it.

Reply:
I just used the stackblitz I provided earlier but there is no specific reason for it so I will inline it and remove the `ng-container`